app/models/events/move_cancel.rb

```diff
@@ -0,0 +1,24 @@
+class Events::MoveCancel < GenericEvent
+  attr_writer :cancellation_reason
+
+  EVENTABLE_TYPES = %w[Move].freeze
```

Comment:
Plan is that each event can support one or more different types of eventable. We validate this and give feedback to the client.

Reply:
Our table currently can hold one type of eventable, how will that look when we add multiple eventables?

Reply:
confirmed there is only one per Event 👍